support-frontend/assets/helpers/abTests/__tests__/abtestTest.ts

```diff
@@ -21,840 +18,303 @@ jest.mock('ophan', () => ({
 }));
 
 // ----- Tests ----- //
-const emptySettings = {
-	switches: {
-		experiments: {},
-	},
-	amounts: {
-		GBPCountries: {},
-		UnitedStates: {},
-	},
-};
-
-const acquisitionDataMockTestControl = {
-	componentId: 'acquisitions-subscription-banner',
-	componentType: 'ACQUISITIONS_SUBSCRIPTIONS_BANNER',
-	abTest: {
-		name: 'mockTest',
-		variant: 'control',
-	},
-};
-
-const acquisitionDataMockTestVariant = {
-	componentId: 'acquisitions-subscription-banner',
-	componentType: 'ACQUISITIONS_SUBSCRIPTIONS_BANNER',
-	abTest: {
-		name: 'mockTest',
-		variant: 'variant',
-	},
-};
-
-const acquisitionDataMockTest2Control = {
-	componentId: 'acquisitions-subscription-banner',
-	componentType: 'ACQUISITIONS_SUBSCRIPTIONS_BANNER',
-	abTest: {
-		name: 'mockTest2',
-		variant: 'control',
-	},
-};
-
-const acquisitionDataMockTest2Variant = {
-	componentId: 'acquisitions-subscription-banner',
-	componentType: 'ACQUISITIONS_SUBSCRIPTIONS_BANNER',
-	abTest: {
-		name: 'mockTest2',
-		variant: 'variant',
-	},
-};
-
-const mockTestControl = `/test.html?acquisitionData=${encodeURI(
-	JSON.stringify(acquisitionDataMockTestControl),
-)}`;
-
-const mockTestVariant = `/test.html?acquisitionData=${encodeURI(
-	JSON.stringify(acquisitionDataMockTestVariant),
-)}`;
-
-const mockTest2Control = `/test.html?acquisitionData=${encodeURI(
-	JSON.stringify(acquisitionDataMockTest2Control),
-)}`;
-
-const mockTest2Variant = `/test.html?acquisitionData=${encodeURI(
-	JSON.stringify(acquisitionDataMockTest2Variant),
-)}`;
-
-describe('basic behaviour of init', () => {
+
+describe('init', () => {
 	Object.defineProperty(window, 'matchMedia', {
-		value: jest.fn().mockImplementation(() => ({
+		value: jest.fn().mockReturnValue({
 			matches: false,
-		})),
+		}),
 	});
+
+	// Common arguments to init
+	const mvt = 123456;
+	const country = 'GB';
+	const countryGroupId = GBPCountries;
+	const emptySettings = {
+		switches: {
+			experiments: {},
+		},
+		amounts: {
+			GBPCountries: {},
+			UnitedStates: {},
+		},
+	} as Settings;
+
```

Comment:
I've pulled out all of these common arguments so that the only arguments that are defined in the test cases are actually important to what we're testing